interface/patient_file/summary/rx_left.php

```diff
@@ -18,6 +37,14 @@
 <span><?php xl('Add','e');?></span></a>
 </td>
 </tr>
+<tr>
+<td>
+<?php if($GLOBALS['rx_show_drug-drug']){ ?>
+<a href="<?php echo $GLOBALS['webroot']?>/interface/weno/drug-drug.php"  target='RxRight' class="css_button" onclick="top.restoreSession()">
+<span><?php xl('Drug-Drug', 'e');?></span></a>
+<?php } ?>
```

Comment:
I took the liberty to go ahead and remove all the e's and change all the xl to xlt.